pkg/controller/kogitodataindex/resource/defaults.go

```diff
@@ -52,6 +52,8 @@ const (
 
 	protoBufKeyFolder string = "KOGITO_PROTOBUF_FOLDER"
 	protoBufKeyWatch  string = "KOGITO_PROTOBUF_WATCH"
+
+	DataIndexEnvKeyHTTPPort = "KOGITO_DATA_INDEX_HTTP_PORT"
```

Comment:
Why is this one public?

Reply:
to be able to use in the tests. should I make it private and use the var name on the test instead?